build/changelog/ChangelogBuilder.php

```diff
@@ -104,6 +104,10 @@ private function createTag($changelogFile)
 
     private function createChangelogJson($changelog, $tag)
     {
+        if (preg_match('/\.\.|\/|\\\\/', $tag)) {
```

Comment:
Could this validation be a little more restrictive. For example a tag value, as far as I am aware of, will be 3 integers separated by a dot "." hence a more restrictive regex would be: `^\d+\.\d+\.\d+$`.

What do you think?

Reply:
I think that's more appropriate.  Updated